core/controller/src/main/scala/whisk/core/loadBalancer/ShardingContainerPoolBalancer.scala

```diff
@@ -71,6 +72,10 @@ class ShardingContainerPoolBalancer(config: WhiskConfig, controllerInstance: Ins
   /** State needed for scheduling. */
   private val schedulingState = ShardingContainerPoolBalancerState()()
 
+  actorSystem.scheduler.schedule(0.seconds, 10.seconds) {
```

Comment:
It might be better to use 60s interval on par with the statsd interval due to the way kamon ingests the metrics to statsd. It sends both histogram and gauges as timers, which forces our statsd plugin to take an average over all received values since the last push which might cause some noise in the resulting metric value, e.g. result in non-integer values after rounding.

Reply:
We took the 10 seconds interval by purpose.
This is a value that changes every few milliseconds (depending how long the actions run). By sending several values and taking an average, the value, that will be displayed in a Dashboard later is much more accurate than a single snapshot which is taken only once a minute.